docs/internal.md

```diff
@@ -151,6 +151,7 @@ The volume label is "cidata", as defined by [cloud-init NoCloud](https://cloudin
 - `LIMA_CIDATA_MNT`: the mount point of the disk. `/mnt/lima-cidata`.
 - `LIMA_CIDATA_USER`: the user name string
 - `LIMA_CIDATA_UID`: the numeric UID
+- `LIMA_CIDATA_HOME`: the guest home directory
```

Comment:
Maybe:
```suggestion
- `LIMA_CIDATA_USER_HOME`: the guest home directory
```

Because we may potentially use `LIMA_CIDATA_HOME` for something like `LIMA_HOME` which is not a user home directory?

Reply:
It was supposed to be simple wrappers for $USER and $HOME, we can give them long and strange names like USER_NAME and USER_HOME but I'm not sure it will help? Better do something like LIMA_CIDATA_HOSTHOME (or LIMA_CIDATA_HOST_LIMA_HOME, or whatever it would end up looking like) when needed

Reply:
This is not visible to end users anyway, so I don't have a strong opinion